src/components/Image/index.js

```diff
@@ -22,7 +22,9 @@ function Image(props) {
             return {uri: `${propsSource.uri}?encryptedAuthToken=${encodeURIComponent(authToken)}`};
         }
         return propsSource;
-    }, [propsSource, isAuthTokenRequired, session]);
+
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
We definitely need a comment explaining why we disable the warning

Reply:
Will update.

Reply:
```
// The `session` prop is not necessary, as it triggers image reloading whenever the session changes. Refer to issue #26034 for more details.
```

Method I used: GP4 + "Could you make this comment more proper English?"

Reply:
@cubuspl42 Updated the comment.